app/scripts/lib/channels/notifier.js

```diff
@@ -20,12 +20,12 @@ define(function (require, exports, module) {
     CHANGE_PASSWORD: {
       name: 'fxaccounts:change_password',
       schema: {
-        email: 'String',
-        keyFetchToken: 'String',
-        sessionToken: 'String',
-        uid: 'String',
-        unwrapBKey: 'String',
-        verified: 'Boolean'
+        email: Vat.email().required(),
+        keyFetchToken: Vat.keyFetchToken().optional(),
```

Comment:
Looks like this one and `unwrapBKey` were required in the previous code, is it intentional that they're optional now?